### PR TITLE
Javascript javascript2 week2

### PR DESCRIPTION
**Pull request by dmytriigithub**

### CODE REVIEW

File: javascript/javascript2/week2/WarmupArrayExercises.js [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+// for (let i = 0; i < numbers.length; i++) {
+//   if (numbers[i] % 2 !== 0) {
+//     newNumbers[i] = numbers[i] * 2;
+//   }
+// }
+
+// console.log("The doubled numbers are", newNumbers); // [2, 6]
+
+const newNumbers = numbers.filter(number => number % 2 !== 0).map(number => number *= 2);
```
Comment (SvitlanaBurlaka) [inline javascript/javascript2/week2/WarmupArrayExercises.js:14]: Good job with rewriting the function 😄

File: javascript/javascript2/week2/WarmupArrayExercises.js [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+console.log(newNumbers);
+
+//---------------------------------------------------------------1.2. Working with movies
+
+//------------------------------------------------------------------1, 2
+
```
Comment (SvitlanaBurlaka) [inline javascript/javascript2/week2/WarmupArrayExercises.js:21]: I would recommend you to create a separate file "movies.js" and store there that long array with movies, so you do not have it in one file 😄 

I cannot se the second part of warmup exercise, have you missed it?

I meant this :
Create an array of movies containing the movies with a short title (you define what short means)
Create an array of movie titles with long movie titles.....

File: javascript/javascript2/week2/hyf-bay/main.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+searchInput.oninput = function() {
+    let value = searchInput.value.toLowerCase().trim();
+
+    if (value) {
+        
+        listItems.forEach(item => {
+
+            if (item.firstChild.innerText.toLowerCase().search(value) == -1) {
+                item.classList.add('hide');
```
Comment (SvitlanaBurlaka) [inline javascript/javascript2/week2/hyf-bay/main.js:37]: I like your solution with class "hide" 👍

**Review by SvitlanaBurlaka — APPROVED** [review]
Wow! 
Nice done 👍  Left some comments , make some minor changes 😄 
Keep it up! 🌟

File: javascript/javascript2/week2/WarmupArrayExercises.js (hunk 2 above)
Comment (dmytriigithub) [reply javascript/javascript2/week2/WarmupArrayExercises.js:21]: They are on lines 146 and 147 😊

Comment (SvitlanaBurlaka) [reply javascript/javascript2/week2/WarmupArrayExercises.js:21]: That is why you have to create a separate file with movies array 😄

**Pull request merged**